.gitignore

```diff
@@ -4,6 +4,7 @@
 
 # VS Code
 .vscode/
+.github/ArchitectureTest.md
```

Comment:
Will we modify this later? @alantao912 @farisdurrani

Reply:
I will take that out. I had ArchitectureText.md in for experimental purposes.